frontend/tasks/pages.js

```diff
@@ -32,7 +33,38 @@ gulp.task('pages-experiments', () => {
     .pipe(gulp.dest(config.DEST_PATH + 'experiments'));
 });
 
-gulp.task('pages-compiled', () => {
+gulp.task('pages-legal', () => {
+  return git.clone(
```

Comment:
Do we need to use git in gulp here? Can't we just copy `node_modules/legal-docs/firefox_testpilot_PrivacyNotice` and `node_modules/legal-docs/firefox_testpilot_Terms` to the spots where they used to be linked from?

Reply:
That won't help, because the problem is that under docker, node_modules/legal-docs doesn't exist... I suppose we could figure out why that is instead.

Reply:
And note that it doesn't exist even though the log shows git does clone it. ¯\_(ツ)_/¯

Reply:
Might be easier to fetch just the documents from raw github URLs, e.g.:

https://raw.githubusercontent.com/mozilla/legal-docs/master/firefox_testpilot_PrivacyNotice/en-US.md
https://raw.githubusercontent.com/mozilla/legal-docs/master/firefox_testpilot_PrivacyNotice/de.md
https://raw.githubusercontent.com/mozilla/legal-docs/master/firefox_testpilot_Terms/en-US.md
https://raw.githubusercontent.com/mozilla/legal-docs/master/firefox_testpilot_Terms/de.md

Trying out the docker command on my machine, but it's going really slow on the npm install step.

Reply:
A *little* wary to clone a git repo in the middle of the gulp build process, but don't have a good reason why. This task is also hooked up to file watchers, but looks like they watch the files that come from the git repo, so it's kind of a no-op I guess.

Reply:
Just downloading the actual urls sounds good. I had considered that but didn't know how to do that with gulp.